test/testFunction.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
File is named incorrectly. Files that contain a class (and there should be one class per file) should be named the same as the class they contain. This file should be called FunctionTest.php

Reply:
ok changed it